lib/beaker/host/pswindows/file.rb

```diff
@@ -2,17 +2,29 @@ module PSWindows::File
   include Beaker::CommandFactory
 
   def tmpfile(name)
-    execute("echo C:\\Windows\\Temp\\#{name}.%RANDOM%")
+    execute("echo #{system_temp_path}\\#{name}.%RANDOM%")
   end
 
   def tmpdir(name)
     # generate name
-    tmpdirname = execute("echo C:\\Windows\\Temp\\#{name}.%RANDOM%")
-    # created named dir
+    tmpdirname = tmpfile(name)
+    # create named dir
     execute("md #{tmpdirname}")
     tmpdirname
   end
 
+  # Turns a host file path into one that can be used by BitVise SSH
+  #
+  # @param [String] path Path to be changed
+  #
+  # @return [String] network version of that path
+  def network_path(path)
```

Comment:
changed to `scp_path`.  I agree that doing this inside SCP helpers would be nice, but everywhere Beaker makes these calls, the platforms are dealt with before sending to the SCP calls so that the calls can focus on what they're supposed to do alone, so changing this now would be too big for this issue.

For your comment about bitvise not being powershell, what fact are we supposed to use to determine whether bitvise is in use from beaker?  It seems like template containing `-bv-` is the only distinguishing fact here.  I just want to make sure that's the right thing to do before potentially changing this up.